src/main/java/games/strategy/triplea/ui/MovePanel.java

```diff
@@ -327,19 +329,19 @@ private void sortTransportsToUnload(final List<Unit> transports, final Route rou
   }
 
   private Match<Unit> getMovableMatch(final Route route, final Collection<Unit> units) {
-    final Match.CompositeBuilder<Unit> movableBuilder = Match.newCompositeBuilder();
+    final PredicateBuilder<Unit> movableBuilder = PredicateBuilder.of(Matches.always());
```

Comment:
Would it be worth having an initial builder method that creates a "true" predicate? Something like PredicateBuilder.always() or PredicateBuilder.true()?